.github/scripts/build-images-base-os.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Why do we need two scripts here if they are almost exactly the same?
The whole issue with the previous GHA setup was that it was not DRY, and we have a similar issue here again.
Every time you try to repeat yourself, think whether you can avoid that, hence DRY.

Reply:
I can merge them together but I don't think we can get benefit from this. They have different building methods, so what I can do is to make this building method into a variable. For instance, passing a build script path to it. However, is it really worthwhile to make the script general and complex in order to only support a specialized condition for base-os (Maybe for the future case)? What's the problem of two scripts with partly repeated but more convenience to maintain?

Reply:
Based on what I see the two scripts are almost identical and there are just a few lines that are different, which can be supported by a simple if-else statement. Hence, as of now it doesn't make sense to me to have two almost identical scripts.

Reply:
OK